### DISCUSSION TITLE
Spline Perspective on established data provenance models like W3C PROV-DM

### DISCUSSION THREAD

**Discussion by ASchwad** [AbsaOSS/spline#821] [Ideas]
Hey there,
what is the perspective of Spline regarding established standards for data provenance models like [PROV-DM](https://www.w3.org/TR/2013/REC-prov-dm-20130430/)?

Choosing a RDF-based Graph instead of a Labeled-property graph comes with a lot of specifications to comply to, but I wondered if there was a deliberate reason to not follow the "standard" solution? Would you mind sharing your thoughts on the (dis-)advantages of the standard with respect to Spline?

Thanks again for your work on this great project!

**Comment by wajda** [comment]
Honestly I didn't know about such standard existence. Thank you very much for the tip.
From the first glance, I think it's pretty much doable. Let us have a closer look and if no fundamental mismatch with our model is found we'll likely put it on our roadmap. I'll let you know.
